ste/sender-azureFile.go

```diff
@@ -383,6 +383,11 @@ func (u *azureFileSenderBase) SetFolderProperties() error {
 	return err
 }
 
+func (u *azureFileSenderBase) DirUrlToString() string {
+	dirUrl := (common.FileURLPartsExtension{FileURLParts: azfile.NewFileURLParts(u.dirURL().URL())}).URL()
```

Comment:
The URL() is on FileURLParts, we don't really need to involve FileURLPartsExtension right?

Reply:
Done.